### PR TITLE
W-7228133 Adding the new fields to docs

### PR DESCRIPTION
**Pull request by gupta298**

### CODE REVIEW

**Comment by gupta298** [comment]
Still waiting on Casey to approve the text for these fields.

File: docs/kb/api-version-3/export.md [hunk 1]
```diff
@@ -70,6 +70,8 @@ OR
...
 * **created_before**: (Optional) Selects visitor activities that were created before the specified time. This value must be after the value in `created_after`. If this argument is not specified, then no upper boundary is used in the query, and all data after the `created_after` is returned. The value can be `today`, `yesterday`, `last_7_days`, `this_month`, `last_month`, or a custom time specified in [GNU Date Input Syntax](http://www.gnu.org/software/tar/manual/html_node/Date-input-formats.html) format.
+* **type**: (Optional) Selects visitor activities of the specified types. If this argument is not specified, then all of the visitor acitivies belonging to any type is returned. See a list of available [Visitor Activity Types](../object-field-references.md#visitor-activity-types) in [Visitor Activity](../object-field-references.md#visitor-activity) in [Object Field References](../object-field-references.md).
+* **prospectOnly**: (Optional) Selects only those visitor acitivites assosiated to a prospect. When this field is set to `false`, all visitor acitivites with and without a prospect are returned. The values can be `true` or `false`.
```
Comment (humeansacrifice) [inline docs/kb/api-version-3/export.md]:
```suggestion
* **prospectOnly**: (Optional) Selects only those visitor activities associated with a prospect. When this field is set to `false`, all visitor activities with and without a prospect are returned. The values can be `true` or `false`.
```

**Comment by mkiely** [comment]
These changes look good. It looks like anchors are not set properly for some objects on Object Field References (everything below Tag Object) https://github.com/pardot/api-docs/blob/master/docs/kb/object-field-references.md#tag-object Mind updating the Visitor Activity one so the link works properly? The others can be handled on another ticket unless you want to roll them into this PR

File: docs/kb/api-version-4/export.md [hunk 2]
```diff
@@ -74,6 +74,8 @@ OR
...
 * **created_before**: (Optional) Selects visitor activities that were created before the specified time. This value must be after the value in `created_after`. If this argument is not specified, then no upper boundary is used in the query, and all data after the `created_after` is returned. The value can be `today`, `yesterday`, `last_7_days`, `this_month`, `last_month`, or a custom time specified in [GNU Date Input Syntax](http://www.gnu.org/software/tar/manual/html_node/Date-input-formats.html) format.
+* **type**: (Optional) Selects visitor activities of the specified types. If this argument is not specified, then all of the visitor acitivies belonging to any type is returned. See a list of available [Visitor Activity Types](../object-field-references.md#visitor-activity-types) in [Visitor Activity](../object-field-references.md#visitor-activity) in [Object Field References](../object-field-references.md).
```
Comment (mkiely) [inline docs/kb/api-version-4/export.md]: typo on activities

File: docs/kb/api-version-3/export.md [hunk 3]
```diff
@@ -74,6 +74,8 @@ OR
...
 * **created_before**: (Optional) Selects visitor activities that were created before the specified time. This value must be after the value in `created_after`. If this argument is not specified, then no upper boundary is used in the query, and all data after the `created_after` is returned. The value can be `today`, `yesterday`, `last_7_days`, `this_month`, `last_month`, or a custom time specified in [GNU Date Input Syntax](http://www.gnu.org/software/tar/manual/html_node/Date-input-formats.html) format.
+* **type**: (Optional) Selects visitor activities of the specified types. If this argument is not specified, then all of the visitor acitivies belonging to any type is returned. See a list of available [Visitor Activity Types](../object-field-references.md#visitor-activity-types) in [Visitor Activity](../object-field-references.md#visitor-activity) in [Object Field References](../object-field-references.md).
```
Comment (mkiely) [inline docs/kb/api-version-3/export.md]: typo on activities (acitivies)

**Pull request merged**